### PR TITLE
[VEN-2744]: IL configurations on Zksync

### PR DESCRIPTION
**Pull request by GitGuru7**
## Description

Resolves VEN

### CODE REVIEW

File: src/vip-framework/index.ts [hunk 1]
```diff
@@ -76,6 +76,11 @@ const executeCommand = async (timelock: SignerWithAddress, proposal: Proposal, c
   if (network.zksync && feeData.maxFeePerGas) {
     // Sometimes the gas estimation is wrong with zksync
     txnParams.maxFeePerGas = feeData.maxFeePerGas.mul(15).div(10);
+    // Increase gas limit for pool registry commands in simulations. Sometimes it is estimated too low in complex transactions.
```
Comment (chechu) [inline src/vip-framework/index.ts:79]: Cannot we define this out of the framework? It seems too specific to be included to something that should be as much generic as possible

File: multisig/proposals/zksyncmainnet/vip-004/index.ts [hunk 2]
```diff
@@ -0,0 +1,340 @@
...
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setRewardTokenSpeeds(address[],uint256[],uint256[])", zksyncmainnet.GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setLastRewardingBlock(address[],uint32[],uint32[])", zksyncmainnet.GUARDIAN],
+    },
```
Comment (chechu) [inline multisig/proposals/zksyncmainnet/vip-004/index.ts:104]: The signature `setLastRewardingBlock(address[],uint32[],uint32[])` is not checked anymore.

`setLastRewardingBlocks(address[],uint32[],uint32[])` and `setLastRewardingBlockTimestamps(address[],uint256[],uint256[])` should be used instead

Comment (GitGuru7) [reply multisig/proposals/zksyncmainnet/vip-004/index.ts]: [05c0740](https://github.com/VenusProtocol/vips/pull/377/commits/05c07404f0bca9365e9700754f8e089bde142f76)

**Review by chechu — APPROVED** [review]

**Review by chechu — APPROVED** [review]

**Comment by chechu** [comment]
https://app.venus.io/#/governance/proposal/365?chainId=56

**Pull request merged**